deployer/scripts/hawkular.sh

```diff
@@ -131,6 +131,25 @@ EOF
 EOF
   
   echo
+  echo "Creating the Hawkular Metrics Client Secrets configuration json file"
+  cat > $dir/hawkular-metrics-client-secrets.json <<EOF
+      {
+        "apiVersion": "v1",
+        "kind": "Secret",
+        "metadata":
+        { "name": "hawkular-metrics-client-secrets",
+          "labels": {
+            "metrics-infra": "hawkular-metrics"
+          }
+        },
+        "data":
+        {
+          "hawkular-metrics.client.account": "$(base64 -w 0 $dir/hawkular-metrics-account.json)"
+        }
+      }
+EOF
+
+  echo
```

Comment:
This should be covered by the hawkular-metrics-account secret, right? Why is there a duplication?